spring-integration-mail/src/test/java/org/springframework/integration/mail/ImapMailReceiverTests.java

```diff
@@ -100,31 +110,69 @@
 
 	private AtomicInteger failed;
 
-	private ImapServer imapIdleServer;
+	private GreenMail imapIdleServer;
+	private GreenMailUser user;
```

Comment:
Our preference is to wrap every single class member with blank lines.
Much easier to review 😄